public/scripts.js

```diff
@@ -0,0 +1,81 @@
+const boxArray = [];
+let i = 1;
+
+const onChange = (e, i) => {
+  i = i - 1;
+
+  if (e.target.checked) {
+    const someStringToCheck = e.currentTarget.id;
+    const indexOfSpace = someStringToCheck.indexOf("-");
+    const index = parseInt(
+      someStringToCheck.substr(indexOfSpace + 1, someStringToCheck.length + 1)
+    );
+    document.getElementById(`box-${i}`);
+    let heightValue = document.getElementById(`height-${index}`).value + "px";
+    let WidthValue = document.getElementById(`width-${index}`).value + "px";
+    console.log("checkbox number -" + index);
+    document.getElementById(`box-${index}`).style.width = WidthValue;
+    document.getElementById(`box-${index}`).style.height = heightValue;
+    document.getElementById(`box-${index}`).style.lineHeight = heightValue;
+  } else {
+    const someStringToCheck = e.currentTarget.id;
+    const indexOfSpace = someStringToCheck.indexOf("-");
+    const index = parseInt(
+      someStringToCheck.substr(indexOfSpace + 1, someStringToCheck.length + 1)
+    );
+    document.getElementById(`box-${index}`).style.width = "100px";
+    document.getElementById(`box-${index}`).style.height = "100px";
+    document.getElementById(`box-${index}`).style.lineHeight = "100px";
+  }
+};
+
+function addbox() {
+  let divMainBox = document.createElement("div");
+  divMainBox.id = `mainBox-${i}`;
+  divMainBox.className = "mainBox";
+  let divDtBorder = document.createElement("div");
+  divDtBorder.className = "dtborder";
+  divDtBorder.id = `box-${i}`;
+  divDtBorder.innerText = "Box";
+  let divDtBorder_two = document.createElement("div");
+  divDtBorder_two.className = "dtborder_two";
+  let divDtBorder_three = document.createElement("div");
+  divDtBorder_three.className = "dtborder_three";
+  let divDtBorder_four = document.createElement("div");
```

Comment:
próbuj w jednej konwencje zmienne i stałe pisać. Jeszcze jedna sprawa. let używaj tylko jak będziesz nadpisywać zmienną. Jeżeli nie to zawsze const.